unix-domain-socket/src/main/scala/akka/stream/alpakka/unixdomainsocket/impl/WrappedFlowStageWithMaterializedValue.scala

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.unixdomainsocket.impl
+
+import akka.actor.ActorRef
+import akka.stream._
+import akka.stream.scaladsl._
+import akka.stream.stage._
+
+import scala.util._
+import scala.util.control.NonFatal
+
+class WrappedFlowStageWithMaterializedValue[In, Out, Mat](
```

Comment:
Please provide a comment here on why the stage is required. I’ve no idea at first glance.